string-manipulation-1/src/is-lower-cased.js

```diff
@@ -0,0 +1,10 @@
+/* exported isLowerCased */
+// if all characters in word are lower cased after function, it is true, if not
+// it is false
+function isLowerCased(word) {
+  if (word === word.toLowerCase()) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
In this case, `word === word.toLowerCase()` already gives us either `true` or `false` so you could have just done: `return word === word.toLowerCase()`